<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LabResultReader</class>
 <widget class="QDialog" name="LabResultReader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="CLabResultEquipmentTableView" name="tblItems"/>
     <widget class="CLabResultLogWidget" name="logWidget" native="true"/>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="chkOnlyFromModel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Обрабатывать только отображенные пробы</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="CProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QCheckBox" name="chkRewrite">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Перезаписовать результат</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CLabResultEquipmentTableView</class>
   <extends>QTableView</extends>
   <header>TissueJournal/LabResultEquipmentTable.h</header>
  </customwidget>
  <customwidget>
   <class>CProgressBar</class>
   <extends>QProgressBar</extends>
   <header>library/ProgressBar.h</header>
  </customwidget>
  <customwidget>
   <class>CLabResultLogWidget</class>
   <extends>QWidget</extends>
   <header>TissueJournal/LabResultLogWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tblItems</tabstop>
  <tabstop>chkRewrite</tabstop>
  <tabstop>chkOnlyFromModel</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LabResultReader</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LabResultReader</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
